src/analysis/modals/DeleteDiskChoices.ts

```diff
@@ -72,7 +68,7 @@ export const DeleteDiskChoices = ({
         p({ style: { marginBottom: 0 } }, ['Also deletes your application configuration and cloud compute profile.']),
       ]
     ),
-    Utils.cond(
+    Utils.cond<React.ReactNode>(
```

Comment:
Super nit: import `ReactNode` from `React` instead of using the namespace here.

```suggestion
    Utils.cond<ReactNode>(
```